### ISSUE TITLE
make tool support?

### ISSUE THREAD

**Issue by Xeverous** [lhmouse/mcfgthread#40]
Hey, I have recently found this Windows GCC distribution. The amount of prebuild stuff (boost, openssl, qslite, python and many others) is amazing but I have realized that your distro does not have `make` tool.

Is there any specific reason behind this? The `dmake` does not work with makefiles I have. I would really like to build with `make` (just like with any other Windows GCC distibution) - is there any easy way to workaround this problem?

**Comment by lhmouse** [comment]
Those are native toolchains (compilers, linkers, etc.) while `make` isn't one of them.

To have a working `make` you have to get a POSIX-conforming shell, for example, by installing [MSYS2](https://www.msys2.org/) which is also where I build GCC, but it would be a different abstraction subsystem.

**Comment by Xeverous** [comment]
> To have a working `make` you have to get a POSIX-conforming shell

Somewhat true, somewhat not. I stopped using MSYS2 because its installer/package manager stopped working altogether (see [this](https://superuser.com/questions/1356256/msys2-and-at-least-one-valid-and-enabled-repository-required-for-this-action-to)).

The shell itself is not really necessary, CMake needs only `make` and nothing else for it's "MinGW makefiles"

I have Git Bash but *just* standalone "GNU make for Windows" covers all CMake and IDE needs.

**Comment by lhmouse** [comment]
I have no idea what 'MinGW makefiles' means exactly amongst CMake generators. I presume that it designates the 'mingw32-make' program, which invokes CMD rather than BASH. Yes I used to include this thing, but soon got a number of questions such as 'why I get this or that error using your `make`?' The 'mingw32-make' program can't be used to process Unix makefiles, because CMD will not recognize shell syntax. In order to eliminate such confusion I removed it.
